requirements.txt

```
@@ -2,9 +2,9 @@
 -e .
 
 # Additional requirements for running the testsuite and development
-pycodestyle==2.2.0
-flake8==3.2.0
+flake8==3.7.9
 pytest==3.7.4
 pytest-cov==2.5.1
 
-Shapely>=1.3.0
+#Shapely>=1.3.0
+https://github.com/Toblerity/Shapely/archive/9ca22240d3272df9598c30878c6f8562e50f1ec8.tar.gz
```

```suggestion
Shapely>=1.3.0, <1.7
```
I think we could just not use Shapely 1.7 in the CI for now (Shapely is not a mandatory dependency of `geoalchemy2`). Anyway, the `to_shape` and `from_shape` functions only use very basic IO features of Shapely which are very stable.